### ISSUE TITLE
Mindustry depends on non existing libs

### ISSUE THREAD

**Issue by leiserfg** [NixOS/nixpkgs#167915]
### Describe the bug
Mindustry tries to load `libasound_module_pcm_pulse.so` but it's no there.

### Steps To Reproduce
Install and run mindustry in a terminal.
An error like

```console
ALSA lib dlmisc.c:337:(snd_dlobj_cache_get0) Cannot open shared library libasound_module_pcm_pulse.so (/nix/store/hjh8ldi38i64b35zkf5bmpqj2zxs0ybw-alsa-lib-1.2.6.1/lib/alsa-lib/libasound_module_pcm_pulse.so: cannot open shared object file: No such file or directory)
```
I checked the `...w-alsa-lib-1.2.6.1/lib/` and it exists but it has not an `alsa-lib/` subfolder
### Expected behavior
Audio works, no errors in the terminal.

### Notify maintainers
@fgaz  @petabyteboy
 
### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.31-2-MANJARO, Manjaro Linux, noversion, rolling`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - channels(leiserfg): `"home-manager, nixpkgs"`
 - channels(root): `""`
 - nixpkgs: `/home/leiserfg/.nix-defexpr/channels/nixpkgs`
```

**Comment by leiserfg** [comment]
Seems like the lib is actually in `pkgs.alsa-plugins` and not in `alsa-lib` so exporting `ALSA_PLUGIN_DIR` in the launcher script should work but I'm still learning nix so I have no idea how to do it.

**Comment by fgaz** [comment]
Fixed in #168004
